src/node_config.cc

```diff
@@ -65,8 +65,18 @@ static void InitConfig(Local<Object> target,
   if (config_preserve_symlinks)
     READONLY_BOOLEAN_PROPERTY("preserveSymlinks");
 
-  if (config_experimental_modules)
+  if (config_experimental_modules) {
     READONLY_BOOLEAN_PROPERTY("experimentalModules");
+    if (!config_userland_loader.empty()) {
```

Comment:
thank you for using `node_config` to expose this :-)

Reply:
That is all @bmeck!